### ISSUE TITLE
Add qualifiers to all imports in some form or another

### ISSUE THREAD

**Issue by VoR0220** [eris-ltd/eris-db#475]
It's incredibly confusing reading through the code and not being able to determine where a variable is being imported from (usually from the Tendermint side of things). It helps very much with clarifying where to find definitions of the code and I think there should be efforts in one form or another to add them in. Furthermore, it helps with open source contributors to know wtf they are doing as well. If there are equally named packages, by all means, throw a well named alias in there. But please get all `.` imports out of the code. 

tl;dr No more `.` aliasing of imports.

**Comment by benjaminbollen** [comment]
On all new and refactored code this coding style has been and will be observed.

**Comment by VoR0220** [comment]
Okay...but there is still code that exists with this. Would you like me to help you clean it?
